cmd/storage-rest-server.go

```diff
@@ -132,6 +133,34 @@ func (s *storageRESTServer) DiskInfoHandler(w http.ResponseWriter, r *http.Reque
 	gob.NewEncoder(w).Encode(info)
 }
 
+func (s *storageRESTServer) UpdateBloomFilter(w http.ResponseWriter, r *http.Request) {
+	if !s.IsValid(w, r) {
+		return
+	}
+	w.Header().Set(xhttp.ContentType, "text/event-stream")
+	var req bloomFilterRequest
+	dec := json.NewDecoder(r.Body)
```

Comment:
we would use gob instead of json as we used gob for almost everything internode communication, have we used json anywhere?

Reply:
Dunno. There is so much magic with gob and I don't like using stuff I don't fully know. But I'll change it.